xml/System.Runtime.InteropServices/DefaultParameterValueAttribute.xml

```diff
@@ -64,16 +64,20 @@
 ## Remarks  
  The <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> attribute allows you to specify a default parameter value in a language that does not otherwise support default parameters.  After you apply this attribute to your code, languages that support default parameters can use the specified value as a default parameter.  
   
- Note that the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> does not add support for default parameters to languages that do not support this feature.  For example, if you use the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> with a method written in C#, which does not support default parameters, you cannot use the default parameter when calling the method from C#.  You can use the default parameter only when calling the method in a language such as C++ that does support the feature.  
+ Note that the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> does not add support for default parameters to languages that do not support this feature.  For example, if you use the <xref:System.Runtime.InteropServices.DefaultParameterValueAttribute> with a language that does not support default parameters, you cannot use the default parameter when calling the method in said language.  You can use the default parameter only when calling the method in a language such as C++ that does support the feature.  
```

Comment:
I think all the .NET languages support it, so I'd suggest removing this paragraph.